pkg/sdk/go/groups.go

```diff
@@ -158,51 +158,51 @@ func (sdk mfSDK) Members(groupID, token string, offset, limit uint64) (auth.Memb
 	return tp, nil
 }
 
-func (sdk mfSDK) Groups(offset, limit uint64, token string) (auth.GroupPage, error) {
-	url := fmt.Sprintf("%s/%s?offset=%d&limit=%d&tree=false", sdk.authURL, groupsEndpoint, offset, limit)
+func (sdk mfSDK) Groups(offset, limit uint64, token string) (GroupsPage, error) {
+	url := fmt.Sprintf("%s/%s?offset=%d&limit=%d&tree=false", sdk.groupsURL, groupsEndpoint, offset, limit)
 	return sdk.getGroups(token, url)
 }
 
-func (sdk mfSDK) Parents(id string, offset, limit uint64, token string) (auth.GroupPage, error) {
-	url := fmt.Sprintf("%s/%s/%s/parents?offset=%d&limit=%d&tree=false&level=%d", sdk.authURL, groupsEndpoint, id, offset, limit, auth.MaxLevel)
+func (sdk mfSDK) Parents(id string, offset, limit uint64, token string) (GroupsPage, error) {
```

Comment:
Using a Groups page, as defined in SDK, is OK (i.e. we redefined structs apparently in SDK for Users, Things, Groups, etc...)